Tests/KodableTests.swift

```diff
@@ -45,6 +45,19 @@ final class KodableTests: XCTestCase {
         }
     }
 
+    func testEncodingNullValuesOutput() throws {
+        struct Strings: Kodable {
+            @Coding var optionalString: String?
+            @Coding(encodeAsNullIfNil: true) var nullOptionalString: String?
+        }
+
+        let strings = Strings()
+        let data = try strings.encodeJSON()
+        let dic = try JSONSerialization.jsonObject(with: data, options: []) as? [String: Any]
+        XCTAssertNil(dic!["optionalString"])
```

Comment:
Perhaps check if `dic!.keys.contains("optionalString") == false`? These 2 comparisons are probably equivalent in Swift, but would be a bit more explicit imo. It's probably just a matter of preference haha

Reply:
I preferred this one because it is less verbose but I guess you have a point ahah